email.md

```diff
@@ -0,0 +1,20 @@
+> just a WIP to get started
+
+## Possible Goals?
+
+- Inform companies/recruiters about what kinds of companies we are looking for (in this case more support for open source)
+- Clear up possible misunderstandings on what open source is/means, how they are looking for talent
+  - Having commits or not doesn't mean much: https://twitter.com/aweary/status/926908058076299264
+  - Having 1 repo in a language/framework doesn't mean I want to work in that language or care
+  - etc
+- Reminder recruiters of economic development benefits (and therefore economic benefits) of open source
+- Explain ways that companies that support open source and why that is a positive perk that we are going to be looking out for
+  - Donations to projects that are used (instead of just saying "we use Babel, JSX, ES6, etc", wouldn't it be amazing to see "we have been sponsoring Babel with $10k/month for the last year because we care about supporting projects that we fundamentally rely on"
+  - Developer time: "We contribute to open source" vs. "we allow our developers to: spend 20% time, or 1 day a week, or weekly hackathons, or employ a maintainer of Babel"
+  - Monthly donation stipend: "we allow our developers to distribute $50 a month to the open source project of their liking"
+
+## Email Response
+
+Thanks for reaching out! I'd be more interested in a company that supports the open source community more: whether it be donating to important projects that the company uses (like on opencollective.com), contributing developer time during work hours to fix bugs, report issues, even maintain a project, or even doing a matching donation similar to a non-profit (maybe give $50 a month to employees for them to choose). If you have any questions about that I'd be happy to explain.
```

Comment:
That sounds like you could still be interested in the offer. What if the recruiter says yes?